src/_locales/en_US/messages.json

```diff
@@ -423,6 +423,18 @@
         "message": "Please add a valid domain or URL",
         "description": "Error text for the form on the Disabled Sites options page tab"
     },
+    "options_disable_tip_title": {
+        "message": "How to disable Privacy Badger without leaving the site",
+        "description": "Title of tip at the top of the Disabled Sites options page tab"
+    },
+    "options_disable_tip_content": {
+        "message": "If you think Privacy Badger is breaking a site you're on, you can click Privacy Badger's button in your browser toolbar, then click the 'Disable for this site' button.",
+        "description": "Content of the tip on the Disabled Sites tab of the Options page. Click options_disable_tip_title to show or hide this content."
+    },
+    "options_disable_tip_alt": {
+        "message": "A screenshot demonstrating how to open the Privacy Badger popup and disable it for a site. Step 1 highlights the Privacy Badger icon in the browser toolbar. Step 2 highlights the disable button on the popup.",
```

Comment:
I think we can shorten this to just the relevant bits. Something like,

>First, click on Privacy Badger's icon in your browser toolbar. Then, click the "Disable for this site" button.

Reply:
"Disable for this site" should be a placeholder probably, see `intro_pin_nudge` for example.